docs/_embed/partner_api.md

```diff
@@ -365,115 +394,131 @@ Get all Zaps in the user's account that have a particular Trello board (assuming
 curl -H "Authorization: Bearer {token}" -L "https://api.zapier.com/v1/zaps?&params__board=BOARD_ID"
 ```
 
+Get all Zaps in the user's account and include their associated param key/value pairs (assuming the OAuth app is Trello).
```

Comment:
```suggestion
Get all Zaps in the user's account and include Trello steps' `params` key/value pairs (assuming the OAuth app is Trello).
```

Small suggestion for clarity. Otherwise, I think this is good to go (whenever @rjpowell says to release)!